src/utils/futil.js

```diff
@@ -38,3 +38,9 @@ export let expandObject = _.curry((transform, obj) => ({
 export let expandObjectBy = _.curry((key, fn, obj) =>
   expandObject(getWith(fn, key))(obj)
 )
+
+export let aspectWrapper = open => F.aspect({
```

Comment:
I think we can simplify this even further if we don't make the wrapper aware of `open` - I don't think end users are going to want to control the open-ness from the aspect

Reply:
but testing this idea here was really smart 👏

Reply:
Changes done.